chromium_src/components/omnibox/common/omnibox_features.cc

```diff
@@ -13,6 +13,8 @@ OVERRIDE_FEATURE_DEFAULT_STATES({{
     {kInspireMe, base::FEATURE_DISABLED_BY_DEFAULT},
     {kDocumentProviderNoSetting, base::FEATURE_DISABLED_BY_DEFAULT},
     {kDocumentProviderNoSyncRequirement, base::FEATURE_DISABLED_BY_DEFAULT},
+    {kExpandedStateHeight, base::FEATURE_DISABLED_BY_DEFAULT},
+    {kExpandedStateShape, base::FEATURE_DISABLED_BY_DEFAULT},
```

Comment:
LGTM! Could you also, please, add them to `BraveMainDelegateBrowserTest.DisabledFeatures` browser test. We try to keep all disabled features in there.

Reply:
Done! (I think?)